.readthedocs.yml

```diff
@@ -19,7 +19,7 @@ formats:
 
 # Optionally set the version of Python and requirements required to build your docs
 python:
-  version: 3.6
+  version: 3.7
```

Comment:
There were a few instances where we were still using 3.6, so I updated it to 3.7 across the board.